### PR TITLE
Add skills section in Home page

### PR DESCRIPTION
**Pull request by nimesh631**
### What does this PR do?
This PR adds a new "Skills" section to the homepage, displaying the front-end, back-end,  essential skills and soft skills. The section includes icons for each skill and organizes them into four categories.

### How to test
1. Pull the branch `feature/skills-section` from the remote repository.
2. Navigate to the homepage and verify that the "Skills" section appears correctly.
3. Ensure that the skills are displayed under the correct categories (Front-End, Back-End, Essential).
4. Check that the icons are visible next to the skill names.

### Additional notes
- The section is designed to be responsive using Tailwind CSS.
- Ensure that all links and navigation in the header are working correctly after merging.

### CODE REVIEW

**Comment by Aayush850** [comment]
### Feedback for **Skills** Section:
- **Color Scheme:** The orange text color (`text-orange-500`) does not match well with the background. Consider using a more neutral color.
- **Icon and Text Spacing:** The icons and their corresponding text in the `<li>` elements are too close. Add some spacing between the icon and the text

**Review by Aayush850 — CHANGES_REQUESTED** [review]
### Feedback for **Skills** Section:
- **Color Scheme:** The orange text color (`text-orange-500`) does not match well with the background. Consider using a more neutral color.
- **Icon and Text Spacing:** The icons and their corresponding text in the `<li>` elements are too close. Add some spacing between the icon and the text

**Review by Himalf — CHANGES_REQUESTED** [review]
### Review for Skills Component Code

#### **Visual Enhancements**:
- The background color (`bg-black bg-opacity-30`) and text color (`text-orange-500`) may be visually confusing, creating readability issues. Consider choosing a higher contrast color scheme for better clarity.
- The spacing between sections could be improved to provide more breathing room and better organization.

#### **Suggestions**:
1. **Add Section Boxes**: Wrapping each skills section in a box with padding and a subtle shadow effect will enhance the visual separation of content.
2. **Improve Gaps**: Increase the gap between sections and items for better readability.

Overall, a good start, but improvements are needed for a more polished, user-friendly design.

File: src/Components/Skills/Skills.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+function Skills() {
+  return (
+    <div>
+      <section id="Skills" className='bg-black bg-opacity-30 shadow-lg rounded-2xl p-5 text-orange-500 '>
```
Comment (Himalf) [inline src/Components/Skills/Skills.jsx]: **Improve this background color.**

File: src/Components/Skills/Skills.jsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            <ul>
+              <li className='flex items-center py-1'>
+                <span className='bg-blue-800 rounded-md'>✔️</span> Team Collaboration
+              </li>
+              <li className='flex items-center py-1'>
+                <span className='bg-blue-200 rounded-md'>✔️</span> Communication
+              </li>
+
+              <li className='flex items-center py-1'>
+                <span className='bg-blue-500 rounded-md'>✔️</span> Problem Solving
```
Comment (Himalf) [inline src/Components/Skills/Skills.jsx]: **In each icons add its appropriate background color.**

**Review by Himalf — CHANGES_REQUESTED** [review]
### Review for Skills Component Code

#### **Visual Enhancements**:
- The background color (`bg-black bg-opacity-30`) and text color (`text-orange-500`) may be visually confusing, creating readability issues. Consider choosing a higher contrast color scheme for better clarity.
- The spacing between sections could be improved to provide more breathing room and better organization.

#### **Suggestions**:
1. **Add Section Boxes**: Wrapping each skills section in a box with padding and a subtle shadow effect will enhance the visual separation of content.
2. **Improve Gaps**: Increase the gap between sections and items for better readability.

Overall, a good start, but improvements are needed for a more polished, user-friendly design.

**Review by Sushant203 — APPROVED** [review]
## feedback about skills section
- The CSS related issue has to be look carefully.
Other  thing are good.

File: src/Components/Skills/Skills.jsx (hunk 1 above)
Comment (nimesh631) [reply src/Components/Skills/Skills.jsx]: Hi  Himalf, I have implemented the changes as per your suggestions:

Updated the color scheme to a more neutral color.
Added spacing between the icons and text in the <li> elements.
Could you please review the updates and let me know if further adjustments are needed? Thanks!"

File: src/Components/Skills/Skills.jsx (hunk 2 above)
Comment (nimesh631) [reply src/Components/Skills/Skills.jsx]: Hi Himalf, I have implemented the changes as per your suggestions:

Updated the color scheme to a more neutral color.
Added spacing between the icons and text in the
elements.
Could you please review the updates and let me know if further adjustments are needed? Thanks!

**Review by Himalf — COMMENTED** [review]
**Requsted updates have been solved.**

**Review by Aayush850 — APPROVED** [review]
All requested changes have been resolved. This is ready to be merged. Approved.

**Review by Himalf — APPROVED** [review]
**Requested update have been solved.**

File: src/Components/Skills/Skills.jsx (hunk 1 above)
Comment (Himalf) [reply src/Components/Skills/Skills.jsx]: **Ok nimesh631 I have reviewed your changes**

**Pull request merged**